app/helpers/products_helper.rb

```diff
@@ -47,13 +47,15 @@ def set_sorting(test, test_status)
 
     case test_status
     when 'passing'
-      return 5
+      return 6
```

Comment:
Yay magic numbers. I understand these were here before, but if we ever go through and clean up code, we should replace these with constants or something.